src/App.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import Header from './components/header/Header';
+import Main from './components/main/Main';
```

Comment:
By default we should create components folder with uppercase letters
You can create a file called `index.js` in those folders to shortcut the import

```suggestion
import Header from './components/Header';
import Main from './components/Main';
```